backend/Cargo.toml

```diff
@@ -5,10 +5,14 @@ edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[[example]]
```

Comment:
"Have you considered relocating `contract_interface_gen.rs` to the scripts directory and configuring `example` to `bin`? This could be more intuitive.

Reply:
I'll take a look into that, thanks!